src/easyreflectometry/data/data_store.py

```diff
@@ -108,6 +112,10 @@ def __init__(
 
         self._color = None
 
+    @property
+    def data_points(self) -> int:
+        return zip(self.x, self.y)
+
```

Comment:
why is it a `property`? Can't we just have it as a standard method?

Reply:
changed to method